src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck1 = new Truck();
+        Machine buldozer1 = new Bulldozer();
+        Machine excavator1 = new Excavator();
 
+        Machine[] machines = {truck1,buldozer1,excavator1};
```

Comment:
don't forget about spaces between variables 
```suggestion
        Machine[] machines = {truck, bulldozer, excavator};
```